### ISSUE TITLE
no-curly-component-invocation gives false positive on yield helper with arguments

### ISSUE THREAD

**Issue by wagenet** [ember-template-lint/ember-template-lint#1089]
In the following template, the first `{{yield}}` is allowed, the second is not.

```hbs
{{#if this.isEnabled}}
  {{yield}}
{{else}}
  {{yield to="inverse"}}
{{/if}}
```

I will try to create a failing test soon.

**Comment by rwjblue** [comment]
Thank you for reporting @wagenet!
